machine-learning/_posts/2018-08-08-HAMLET.md

```diff
@@ -0,0 +1,213 @@
+        Diffusion MRI"
+tags: deep-learning tractography
+author: "Daniel Jörgens"
+cite:
+    authors: "Reisert, M and Coenen, V A and Kaller, C and Egger, K and
+              Skibbe, H"
+    title:   "HAMLET: Hierarchical Harmonic Filters for Learning Tracts
+              from Diffusion MRI"
+    venue:   "arvix.org"
+    year:    "2018"
+pdf: "https://arxiv.org/pdf/1807.01068.pdf"
+---
+
+
+# Contribution
+
+The authors propose a new set of learnable filters for extracting
+both **location** and **orientation** of a specific white matter bundle
+in the human brain. In that sense, their proposal combines the ideas
+of bundle specific tracking and pure bundle segmentation.
+
+As opposed to common deep learning approaches, their filter needs only
+a few parameters to map 12 chosen bundles (1e3 for HAMLET vs 1e6 for common
+CNN implementations in other tasks).
+
+Further, their approach makes use of covariance properties of the data,
+i.e. translation as well as rotation covariance.
+
+These two factors enable their filter to be trained with a relatively
+modest number of training samples.
+
+# Overview
+
+<p style="text-align:center"><img src="/machine-learning/images/HAMLET/hamlet_overview.png" width="700"></p>
+
+# Covariant Non-linear filters based on Spherical Tensor Algebra
+
+#### Prereqisites
+
+ * **Spherical tensor fields**
+
+   $$x(\mathbf{r}, \mathbf{n}) = \sum_{j=0}^\infty \mathbf{a}^j(\mathbf{r})^\mathsf{T}
+                                                  \mathbf{Y}^j(\mathbf{n})$$.
+
+  * $$\mathbf{Y}^j: S_2 \mapsto \mathbb{C}^{2j+1}$$ are SH basis functions
+    of order *j*.
+  * $$\mathbf{a}^j: \mathbb{R}^3 \mapsto \mathbb{C}^{2j+1}$$ is a
+    *spherical tensor field*.
+
+ * **Spherical product**
+
+   The authors define a spherical product
+   $$\circ_j : \mathbb{C}^{2j_1+1} \times \mathbb{C}^{2j_2+1}
+     \mapsto \mathbb{C}^{2j+1}$$
+   which is ***rotationally covariant*** in the sense of
+
+   $$(\mathbf{D}^{j_1}(g)\mathbf{v}) \circ_j (\mathbf{D}^{j_2}(g)
+     \mathbf{w}) = \mathbf{D}^j(g)(\mathbf{v} \circ_j \mathbf{w})$$,
+
+   where $$\mathbf{v} \in \mathbb{C}^{2j_1+1}$$,
+         $$\mathbf{w} \in \mathbb{C}^{2j_2+1}$$ and
+         $$g \in SO(3)$$ and
+         $$\mathbf{D}^j(g)$$ is the *Wigner-D* matrix.
+
+ * **Spherical differential operator**
+
+   Given the *solid harmonics* defined by the homogeneous polynomials
+   $$R_m^j(\mathbf{r}) = r^jY^j_m(\mathbf{n})$$ (where $$r = |\mathbf{r}|$$
+   and $$\mathbf{n} = \frac{\mathbf{r}}{|\mathbf{r}|}$$), they further
+   define a *spherical differential operator*
+
+   $$\mathbf{\partial}^j := (\partial^j_{-j}, ..., \partial^j_{j})$$,
+
+   where $$\partial^j_m := R^j_m(\nabla)$$ on the basis of spherical
+   products.
+
+   The authors emphasise that this *"operator can be used to linearly
+   map" spherical tensor fields "onto other" spherical tensor fields
+   "by intertwining spatial neighbourhood information"*.
```

Comment:
neighborhood, no "u"

Reply:
I try to use British English. Is that OK?